plugins/Table2/scripts/Table.js

```diff
@@ -0,0 +1,37 @@
+const { ContentTab, openTab } = await require('@bridge/tab')
+const { create } = await require('@bridge/sidebar')
+const { Table2 } = await require('@bridge/ui')
+const { register, addTabActions } = await require('@bridge/tab-actions')
+const { setup } = await require('@bridge/com-mojang')
+
+
+class Table2Tab extends ContentTab {
+    type = 'Table2Tab'
+    component = Table2
+
+	async setup() {
```

Comment:
You can remove the `setup()` function here because you don't use tab actions & the com.mojang directory doesn't need to be set afaik